.flake8

```diff
@@ -0,0 +1,81 @@
+[flake8]
```

Comment:
No need to copy and paste. Remove unnecessary rules.

Reply:
How so? :( Removed.